LayoutTests/inspector/console/js-completions.html

```diff
@@ -52,6 +52,27 @@
         }
     });
 
+    suite.addTestCase({
+        name: "console.jsCompletions.completionOrdering",
+        description: "Test that the JavaScript completions are ordered correctly.",
+        async test() {
+            await InspectorTest.evaluateInPage(`myObject = {
+                1: 42, 2: 42, 10: 42, 20: 42, aa: 42, ab: 42, Aa: 42, Ab: 42, __aa__: 42, __ab__: 42
+            };`);
+            WI.javaScriptRuntimeCompletionProvider.clearCachedPropertyNames();
+
+            InspectorTest.json(await generateJSCompletions({baseString: "myObject.", prefix: ""}));
+            InspectorTest.json(await generateJSCompletions({baseString: "myObject.", prefix: "a"}));
+            InspectorTest.json(await generateJSCompletions({baseString: "myObject.", prefix: "A"}));
```

Comment:
should we do a few of these same tests again with `WI.settings.experimentalShowCaseSensitiveAutocomplete` flipped?

Reply:
what makes you say for sure that we'll end up removing it?  we've had settings for to control case-sensitive search for years.  i personally would strongly prefer keeping it case-sensitive, but i realize that's my preference so i'm fine with adding a way for others to have a different experience

also, if we do end up removing it, until we do we should test it :)